### PR TITLE
VTC_005: [BE] Implement Visits BL

### PR DESCRIPTION
**Pull request by Ghostyshka**

### CODE REVIEW

File: VetClinicManagament/Core/Domain/Entities/InvoiceItem.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 public class InvoiceItem
 {
     public int ItemId { get; set; }
-    public int InvoiceId { get; set; }
+    public int? InvoiceId { get; set; }
```
Comment (denyshfive) [inline VetClinicManagament/Core/Domain/Entities/InvoiceItem.cs:6]: why it's nullable?

File: VetClinicManagament/Core/Domain/Models/Dtos/InvoiceDto.cs [hunk 2]
```diff
@@ -8,5 +8,5 @@ public class InvoiceDto
     public DateTime CreatedAt { get; set; }
     public DateTime UpdateAt { get; set; }
     public InvoiceStatus InvoiceStatus { get; set; }
-    public List<InvoiceItemDto> InvoiceItems { get; set; }
+    public List<int> InvoiceItems { get; set; }
```
Comment (denyshfive) [inline VetClinicManagament/Core/Domain/Models/Dtos/InvoiceDto.cs:11]: why did you change it?

File: VetClinicManagament/Core/Domain/Models/Dtos/InvoiceItemDto.cs [hunk 3]
```diff
@@ -2,7 +2,6 @@
 
 public class InvoiceItemDto
 {
-    public int ItemId { get; set; }
-    public int InvoiceId { get; set; }
+    public int? InvoiceId { get; set; }
```
Comment (denyshfive) [inline VetClinicManagament/Core/Domain/Models/Dtos/InvoiceItemDto.cs:5]: how invoice item can be without invoice?

File: VetClinicManagament/Core/Domain/Models/Dtos/MedicalDto.cs [hunk 4]
```diff
@@ -1,8 +1,8 @@
-﻿namespace Domain.Entities;
+﻿namespace Domain.Models.Dtos;
 
-public class Medicals
+public class MedicalDto
 {
-    public int MedicalId { get; set; }
+    public int? MedicalId { get; set; }
```
Comment (denyshfive) [inline VetClinicManagament/Core/Domain/Models/Dtos/MedicalDto.cs:5]: why id is nullable?

**Review by denyshfive — CHANGES_REQUESTED** [review]

File: VetClinicManagament/Core/Domain/Models/Dtos/InvoiceDto.cs (hunk 2 above)
Comment (Ghostyshka) [reply VetClinicManagament/Core/Domain/Models/Dtos/InvoiceDto.cs:11]: returned the previous version, changed it to an int list for easy testing
